README.md

```diff
@@ -1,13 +1,57 @@
-Examples of player applications on EXP. See the [documentation](https://docs.goexp.io).
+# Exp App Boilerplates
 
-These app template boilerplates are the global apps available to all EXP users.
+Examples of player applications on EXP. See the [documentation](https://docs.goexp.io). These app template boilerplates are the global apps available to all EXP users. Some of the apps are hidden from the UI for simplicity.
 
-The following is a list of UUIDs for each global app in EXP.
+# Examples Using Common Frameworks
+
+- angular: [meeting room app](meeting-room-app)
+- vuejs: [news app](news-app)
+- ember: [social app](social-feed)
+- jquery: [menu board app](menu-board)
+
+# Special and Hidden Apps
+
+## Bootloader
+
+The [bootloader app](bootloader) is launched by the player to start playback. Its purpose is to ensure authentication is successful and to either launch the [experience app](#experience) or [pairing app](#pairing) depending on whether or not you are authenticated. You can use a custom app instead of the stock bootloader by using the url query/hash parameter `bootloader=[uuid]`.
+
+## Experience
+
+The [experience app](experience) launches the [menu app](#menu), [identify app](#identify), and [fling app](#fling). It also reads the current experience of the device (or the experience specified in by the url parameter `experience=[uuid]`) and launches the [schedule app](#schedule) with the appropriate configuration.
+
+## Schedule
+
+The [schedule app](schedule) takes a set of recursion rules that map apps to days and plays the appropriate app for the current day (local date).
+
+## Dayplan
+
+The [dayplan app](dayplan) takes a map of apps to time blocks and plays the appropriate app for the current time (local time).
+
+
+## Pairing
+
+The [pairing app](pairing) shows the pairing code on screen and listens for the pairing event. When credentials are received it saves them and restarts the player.
+
+## Menu
+
+The [menu app](menu) shows a menu on screen when the user clicks near the bottom left corner. The menu shows device, experience, and organization name, as well as online/offline status and cache storage.
+
+## Identify
+
+The [identify app](identify) listens for the `identify` event on the device channel and shows the device name on screen for a short period of time when the message is received.
+
+## Fling
+
+The [fling app](fling) listens for events on contextual channels and forwards the event payload to the `exp.player.play()` method.
+
+
+# UUIDs
+
+The following is a list of UUIDs for each global app in EXP. These apps are available publically without credentials.
 
 | App Template | UUID |
 |--------------|------|
 |audio|3415bd0b-f567-4f53-896a-2344b24645be|
-|dayplan|4d0d80e8-fdba-46a2-8a54-90fdab96390d
```

Comment:
To be clear, I copy-pasted the README.md, and this was the state of current develop